rollup/rollup_sync_service/rollup_sync_service_test.go

```diff
@@ -327,7 +391,7 @@ func TestGetChunkRangesCodecv2(t *testing.T) {
 		t.Fatalf("Failed to read RLP data: %v", err)
 	}
 	l1Client := &mockEthClient{
-		commitBatchRLP: rlpData,
+		txRLP: rlpData,
```

Comment:
since there are `commitBatch` transactions and `commitBatchWithBlobProof` transactions, changing this variable's name to a more general name.